README.md

```diff
@@ -70,69 +76,66 @@ Use `grunt serve` for development.
 
 ## Support/Help
 
-- IRC: irc.hackint.org #freifunkRGB
+- IRC: irc.hackint.org #freifunkRGB or development-channel #meshviewer
```

Comment:
maybe a little more structure for the IRC channels and IRC links:

    - IRC on irc.hackint.org
        - [#freifunkRGB](irc://irc.hackint.org/freifunkRGB)
        - [#meshviewer](irc://irc.hackint.org/meshviewer) (development-channel)